install-deps.sh

```diff
@@ -306,6 +308,7 @@ else
                 ensure_decent_gcc_on_ubuntu 8 trusty
                 ;;
             *Xenial*)
+                python_interpreters+=" python2.7"
```

Comment:
I'm in favor of adding this code to the stable version(s) of Ceph that support Xenial. Does that include master/Octopus?

Reply:
master and Octopus are going to be Python3 exclusive. Xenial should work.

Reply:
by "Xenial should work" do you mean it will work with both python3 and python2.7?

Reply:
from the Python perspective, it should work since Xenial is python3. It optionally allows installation of Python2. From the rest of the Ceph dependencies... I don't know.

Master/Octopus will not be Python2/Python3 compatible, the idea is Python3 exclusive

Reply:
FWIW, python3.5 on xenial is not good enough. it's missing some features we are using on our python3 support. see https://github.com/ceph/ceph/pull/31525#issuecomment-553402698. and i don't think we should support python3.5.